frontend/src/components/FacilityBooking/ViewBlock.tsx

```diff
@@ -19,15 +22,24 @@ const ViewBlock = (props: Props) => {
   }, [ref.current])
 
   return (
-    <StyledViewBooking ref={ref} isOccupied={props.entry.type === TimeBlockType.OCCUPIED} blockId={props.entry.id}>
-      {props.entry.type === TimeBlockType.OCCUPIED && (
-        <>
-          {props.entry.ccaName}
-          <br />
-          {props.entry.eventName}
-        </>
+    <>
+      {props.entry.type === TimeBlockType.OCCUPIED ? (
+        <StyledViewBooking
+          ref={ref}
+          isOccupied={true}
+          blockId={props.entry.id}
+          onClick={() => (props.onClickFunction(true), props.setViewBooking(props.entry.booking ?? myBookingsStub[0]))}
+        >
+          <>
```

Comment:
this wrapper is not necessary since StyledViewBooking is the parent

Reply:
fixed in e39d7185d